2024/YoctoProjectDeveloperDay/cfp/practical-security-tee-secure-storage.md

```diff
@@ -0,0 +1,41 @@
+# Title
+
+Practical Security for Embedded Systems: Implementing TEE and Secure Storage
+
+# Abstract
+
+In this presentation, we will explore practical security measures for embedded
+systems using Trusted Execution Environment (TEE) and secure storage. We will
+cover the benefits of TEEs, the structure of a root of trust, and share insights
+from our experience of integrating it within our Yocto-based Secure Embedded
+Linux distribution, Zarhus OS. Attendees will gain practical knowledge and tools
+to enhance the security of their embedded systems.
+
+# Description
+
+In this session, we will delve into the essential aspects of securing embedded
+systems. The session begins with an introduction to the importance of security
+in embedded systems, followed by an explanation of key concepts such as the root
+of trust and the architecture of Trusted Execution Environments (TEEs).
+
+We will then discuss the practical implementation of TEEs and secure storage,
+detailing how they function and the security benefits they provide.
+
+Next, we will move to the current support within Yocto, discussing available
+tools, resources, and ongoing developments that support secure implementations.
+This discussion will be illustrated with our custom Yocto-based
+Secure Embedded Linux distribution, Zarhus OS. We will explain its design, the
+security features it incorporates, and how it enhances the security of embedded
+systems. This segment will provide practical insights into the development and
+deployment of a secure embedded
+Linux distribution.
+
+The session will include a case study of our involvement in the CROSSCON
+project, an initiative aimed at standardizing the IoT security stack, and the
+problems it tackles.
+
+By the end of this session, participants will have a comprehensive understanding
+of TEEs and secure storage, equipped with the knowledge and tools to implement
+these security measures in their own embedded systems. They will also gain
+insights from practical examples and ongoing projects to aid in their security
+implementations.
```

Comment:
It is too much LLM-driven exaggeration. Please be humble. Words like "comprehensive" are high red light.

Please try to think like this:
- for what market segment can this feature be used?
- How can this feature be used in that market segment?

Reply:
Added a bit from my side here 5529023f6ba41a84dc135e5e0fa6d26591fd9fa9.

Reply:
It is much better, we don't have time to polish it more, so I would say it is good to go.